packages/docs-markdown/src/controllers/quick-pick-menu-controller.ts

```diff
@@ -173,6 +173,10 @@ export function markdownQuickPick(context: ExtensionContext) {
 		{
 			description: '',
 			label: '$(lock) Non-localizable text'
+		},
+		{
+			description: '',
+			label: '$(browser) Yaml preview'
```

Comment:
When I navigate to a yaml or markdown file I dont see this option. Can you check that this is working correctly? We only want to show this Yaml preview when the user is in a .yml file.

Reply:
I just tested , its showing on my end , not sure what cause it.
![1](https://user-images.githubusercontent.com/36059024/96670900-5ec2d580-1315-11eb-995c-8a8c64a9733e.png)
![2](https://user-images.githubusercontent.com/36059024/96670901-5f5b6c00-1315-11eb-85a1-e010a665998b.png)